test/e2e/utils.go

```diff
@@ -586,6 +656,26 @@ func getOtelCollectorOptions() map[string]interface{} {
 	return otelOptions
 }
 
+func getOtelAgentOptions() map[string]interface{} {
+	otelOptions := map[string]interface{}{
+		"extensions": map[string]interface{}{
+			"health_check": map[string]string{"port": "14269"},
+		},
+	}
+
+	return otelOptions
+}
+
+func getOtelAllInOneOptions() map[string]interface{} {
```

Comment:
This func is the same as the `getOtelAgentOptions` I would unify and use just one `getOTELConfig(healthCheckPort int)`.